src/api/data-migration/version-4-to-version-5/update-applications/update-application-migrated-fields.ts

```diff
@@ -0,0 +1,23 @@
+import { Connection } from 'mysql2/promise';
+import { APPLICATION } from '../../../constants';
+import executeSqlQuery from '../../execute-sql-query';
+
+const { STATUS } = APPLICATION;
+
+/**
+ * updateInProgressApplicationsMigratedTo
+ * Update IN_PROGRESS applications migratedTo fields to have a value of 4.
+ * @param {Connection} connection: SQL database connection
+ * @returns {Promise<object>} executeSqlQuery response
+ */
+const updateInProgressApplicationsMigratedTo = (connection: Connection) => {
+  const loggingMessage = `Updating in progress application's migratedTo FIELD to 5 in the application table`;
+
+  const query = `
+    UPDATE Application SET migratedTo=5 WHERE status = '${STATUS.IN_PROGRESS}'
```

Comment:
We could benefit by declaring `5` inside `version-4-t-version-5` constants directory, not a blocker though.

Reply:
Missed this one.

Reply:
Updated to use existing `LATEST_VERSION_NUMBER` constant.